templates/locales/en.yml

```diff
@@ -1663,11 +1663,22 @@ en:
         4.2.14 contains a critical bug which prevents it from working with
         Vagrant. VirtualBox 4.2.16+ fixes this problem. Please upgrade
         VirtualBox.
+      virtualbox_disks_controller_not_found: |-
+        The current disk configuration requires a controller with storage bus
+        '%{storage_bus}', but no such controller was found. Please add the
+        proper controller to your guest using provider-specific
```

Comment:
We could probably be specific here about the provider since the error is `virtualbox_disks`. Is there any info we could give to make it easier to solve this issue?